zen/setup/environment_detector.py

```
@@ -79,226 +84,219 @@ def detect_environment(self, zenos_root: Path) -> EnvironmentInfo:
-        
+                return "powershell"  # Default to PowerShell
+
         # Try to detect from process
         try:
             import psutil
+
             current_process = psutil.Process()
             parent = current_process.parent()
             if parent:
                 return Path(parent.exe()).name
         except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)